skymp5-server/cpp/addon/main.cc

```diff
@@ -360,6 +364,16 @@ ScampServer::ScampServer(const Napi::CallbackInfo& info)
       }
     }
 
+    if (serverSettings["lang"] == nullptr) {
+      serverSettings["lang"] = "english";
+    }
+
+    logger->info("Run localization provider for language:",
+                 serverSettings["lang"]);
+    this->localizationProvider =
+      std::shared_ptr<LocalizationProvider>(new LocalizationProvider(
+        serverSettings["dataDir"], serverSettings["lang"]));
```

Comment:
It's better to use `std::make_shared` to write less.
```suggestion
    localizationProvider = std::make_shared<LocalizationProvider>(serverSettings["dataDir"], serverSettings["lang"]);
```